src/solve.jl

```diff
@@ -0,0 +1,16 @@
+@inline function (\){T}(A::SMatrix{2,2,T,4}, b::SVector{2,T})
+    d = det(A)
+    @inbounds x = SVector( A[2,2]*b[1] - A[1,2]*b[2],
+                           A[1,1]*b[2] - A[2,1]*b[1] ) / d
+    return x
+end
+
+@inline function (\){T}(A::SMatrix{3,3,T,9}, b::SVector{3,T})
+    d = det(A)
+    @inbounds begin
+        x = SVector( A[1,2]*A[2,3]*b[3] - A[1,2]*A[3,3]*b[2] - A[1,3]*A[2,2]*b[3] + A[1,3]*A[3,2]*b[2] + A[2,2]*A[3,3]*b[1] - A[2,3]*A[3,2]*b[1],
+                    -A[1,1]*A[2,3]*b[3] + A[1,1]*A[3,3]*b[2] + A[1,3]*A[2,1]*b[3] - A[1,3]*A[3,1]*b[2] - A[2,1]*A[3,3]*b[1] + A[2,3]*A[3,1]*b[1],
+                     A[1,1]*A[2,2]*b[3] - A[1,1]*A[3,2]*b[2] - A[1,2]*A[2,1]*b[3] + A[1,2]*A[3,1]*b[2] + A[2,1]*A[3,2]*b[1] - A[2,2]*A[3,1]*b[1] ) / d
```

Comment:
I've changed the version a bit to reduce the number of flops by factoring out some `b`s. Now, this version has the same number of additions and multiplications as a version based on `inv` but this version only has three division whereas the `inv` based version has six and because they are much more expensive I think I see a 2x difference in the 3x3 case.